cinder/manifests/init.pp

```diff
@@ -472,8 +482,19 @@
     cinder_config {
       'DEFAULT/ssl_cert_file' : value => $cert_file;
       'DEFAULT/ssl_key_file' :  value => $key_file;
+      'DEFAULT/nova_endpoint_template': value => $nova_pub_url;
+      'keystone_authtoken/auth_protocol': value => 'https';
+      'keystone_authtoken/auth_uri': value => $keystone_auth_uri;
+      'keystone_authtoken/identity_uri': value => $keystone_identity_uri;
```

Comment:
can we remove auth_uri and identity_uri and check what happens? Ideally, they should not be required.